<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab4ChartPlot</class>
 <widget class="QWidget" name="Tab4ChartPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3,3,1,1">
     <item>
      <widget class="QDateTimeEdit" name="pDateTimeEditTo">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2021</year>
         <month>9</month>
         <day>29</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="pDateTimeEditFrom">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2021</year>
         <month>12</month>
         <day>31</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pLEName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBAddButton">
       <property name="text">
        <string>조회</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBClearButton">
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
